tests/src/GC/Features/SustainedLowLatency/scenario.cs

```diff
@@ -109,6 +109,8 @@ static public void Main(String[] args)
 
             total_elapsed_ms = stopwatch.Elapsed.TotalMilliseconds;
         }
+
+        return 100;
```

Comment:
I guess the failure mode is crashing? (since there are no other "return" statements)